draft-ietf-anima-brski-prm.md

```diff
@@ -2111,44 +2274,86 @@ The pledge MAY provide the dedicated endpoint for the Query Pledge Status operat
+* 406 Not Acceptable: if the Accept request header field indicates a type that is unknown or unsupported, e.g., a type other than `application/jose+json`
+* 415 Unsupported Media Type: if the Content-Type request header field indicates a type that is unknown or unsupported, e.g., a type other than `application/jose+json`
+
+The pledge MAY use the response body to signal failure details to the service technician operating the Registrar-Agent.
+
+
+### Request Artifact: Status Trigger (tStatus) {#tstatus_artifact}
+
+The Status Query (tStatus) artifact SHALL be an authenticated self-contained object signed by the pledge, providing status query parameters.
+
+For the JWS-signed JSON format used by this specification, the tStatus artifact MUST use the "General JWS JSON Serialization Syntax" defined in {{Section 7.2.1 of RFC7515}}, which MUST contain the JSON Status Trigger Data defined in {{tstatus_data}} in the JWS Payload.
+{{tstatus_representation}} summarizes the serialization of the JWS-signed JSON PER artifact:
+
+~~~~
+{
+  "payload": BASE64URL(UTF8(JSON Status Trigger Data)),
+  "signatures": [
+    {
+      "protected": BASE64URL(UTF8(JWS Protected Header)),
+      "signature": BASE64URL(JWS Signature)
+    }
+  ]
+}
+~~~~
+{: #tstatus_representation title='tStatus Representation in General JWS JSON Serialization Syntax' artwork-align="left"}
 
-If the pledge provides the Query Pledge Status endpoint, it MUST reply to this request with the Pledge Status (pStatus) artifact in the body of a 200 OK response.
-The response header MUST have the Content-Type field set to `application/jose+json`.
+The JSON Status Trigger Data MUST be UTF-8 encoded to become the octet-based JWS Payload defined in {{RFC7515}}.
+The JWS Payload is further base64url-encoded to become the string value of the `payload` member as described in {{Section 3.2 of RFC7515}}.
+The octets of the UTF-8 representation of the JWS Protected Header are base64url-encoded to become the string value of the `protected` member.
+The generated JWS Signature is base64url-encoded to become the string value of the `signature` member.
 
-### Request Artifact: Status Trigger (tStatus)
+#### JSON Status Trigger Data {#tstatus_data}
 
-The Status Query artifact is a JWS structure signing information on the requested status-type, the time and date the request is created, and the product serial-number of the pledge contacted as shown in {{stat_req_def}}.
-The following Concise Data Definition Language (CDDL) {{RFC8610}} defines the structure of the unsigned Status Query data (i.e., JWS payload):
+The JSON Status Trigger Data SHALL be a JSON document {{RFC8259}} that MUST that MUST conform with the CDDL {{!RFC8610}} data model defined in {{stat_req_def}}:
 
 ~~~~ cddl
   statustrigger = {
       "version": uint,
-      "created-on": tdate,
       "serial-number": text,
-      "status-type": text
+      "created-on": tdate,
+      "status-type": $status-type
   }
+
+  $status-type /= "bootstrap"
```

Comment:
I added the extension point that was learned for the tPER CDDL definition.
Please check.

Reply:
Was not aware of that option but provides the flexibility to allow other queries